docs/how-to-docs/cert-manager.md

```diff
@@ -141,18 +171,22 @@ $ kubectl -n project get certificate
 NAME        AGE
 example-tls    1d
 
-# you can also review the order and challenge via
+# you can also review the certfificaterequests, orders and challenges via
 $ kubectl -n project get orders
 $ kubectl -n project get challenge
 ```
 
 **Network Policies**
 
-In order to handle the http01 challenge the requestor is provided an ephermal token which must be handled back to LetsEncrypt on the path `http://domain/.well-known/acme-challenge/`, thus validating to them you own the domain's your requesting for. Cert-manager handles the http01 resolver by;
+In order to handle the http01 challenge the requestor is provided an ephermal token which must be handled back to LetsEncrypt on the path `http://domain/.well-known/acme-challenge/`, thus validating to them you own the domain's your requesting for.
```

Comment:
`ephermal` => `ephemeral`

Reply:
`domain's your requesting for.` => `domains you're requesting for.`

Reply:
k